File: sach/de3312.cpp
//DE3312.CPP - Tran Huu Nam - hsgtin.vn - 2024
#include <iostream>
using namespace std;
int main() {
    string s;
    getline(cin,s);
    int dmax = 0,d = 0;
    for(int i=0;i<s.size();i++){
        if(s[i] == '0') {
            d++;
            if (d>dmax) dmax=d;
        }
        else
            d = 0;
    }
    cout<<dmax;
    return 0;
}

File: sach/de3312.py
//DE3312.PY - Tran Huu Nam - hsgtin.vn - 2024
import sys
def main():
    s = sys.stdin.readline().strip()
    dmax = 0
    d = 0
    for i in range(len(s)):
        if s[i] == '0':
            d += 1
            if d > dmax:
                dmax = d
        else:
            d = 0
    print(dmax)
if __name__ == "__main__":
    main()

